Assets/PlayroomKit/modules/Helpers/CommandManager.cs

```diff
@@ -61,13 +61,54 @@ private void RegisterCommands()
             },
             new()
             {
-                Command = "WaitForState", Description = "Waits for state to finish", Callback = WaitForStateCommand
+                Command = "WaitForState", Description = "Waits for state to finish -statekey(string)", Callback = WaitForStateCommand
             },
             new()
             {
                 Command = "OnDisconnect", Description = "Callback on player disconnect", Callback = OnDisconnectCommand
             },
-            new() { Command = "ResetState", Description = "Reset state to default", Callback = ResetStateCommand }
+            new() { Command = "ResetState", Description = "Reset state to default -keyToExclude (string)", Callback = ResetStateCommand },
+
+            new()
+            {
+                Command = "PlayerGetProfile", Description = "Returns player profile", Callback = PlayerGetProfileCommand
+            },
+            
+            new()
+            {
+                Command = "PlayerGetState", Description = "Returns Playrooms Player State  -key(string)", Callback = PlayerGetStateCommand
+            }, 
+            
+            new()
+            {
+                Command = "PlayerSetState", Description = "Sets Playrooms Player State  -key(string) -value(string) -reliable(true/false)", Callback = PlayerSetStateCommand
+            },
+            
+            new ()
+            {
+                Command = "PlayerWaitForState ", Description = "Waits for state of Playrooms Player -stateKey(string) ", Callback = PlayerWaitForStateCommand   
+            },
+            
+            new ()
+            {
+                Command = "KickPlayer", Description = "Kicks PLayer", Callback = KickPlayerCommand  
```

Comment:
Can we have "Player.Kick" instead of "KickPlayer" as a command? That is more realistic.

Reply:
Same for the other commands.